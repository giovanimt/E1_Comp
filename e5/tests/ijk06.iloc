addI rsp, 4 => rsp
addI rsp, 4 => rsp
addI rsp, 4 => rsp
loadI 75 => r0
addI rfp, 4 => r1
store r0 => r1
loadI 3 => r2
addI rfp, 8 => r3
store r2 => r3
addI rfp, 4 => r4
load r4 => r5
addI rfp, 8 => r6
load r6 => r7
div r5, r7 => r8
addI rfp, 0 => r9
store r8 => r9
addI rfp, 0 => r10
load r10 => r11
loadI 5 => r12
div r11, r12 => r13
addI rfp, 4 => r14
store r13 => r14
addI rfp, 0 => r15
load r15 => r16
addI rfp, 4 => r17
load r17 => r18
div r16, r18 => r19
addI rfp, 8 => r20
store r19 => r20
