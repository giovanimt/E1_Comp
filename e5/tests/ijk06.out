00013810 25
00013814 5
00013818 5
